ui-core/src/main/res/values/attrs.xml

```diff
@@ -9,8 +8,15 @@
 
 <resources>
     <declare-styleable name="RoundCornerImageView">
-        <attr name="adyenRadius" format="dimension"/>
-        <attr name="adyenStrokeWidth" format="dimension"/>
-        <attr name="adyenStrokeColor" format="color"/>
+        <attr name="adyenRadius" format="dimension" />
+        <attr name="adyenStrokeWidth" format="dimension" />
+        <attr name="adyenStrokeColor" format="color" />
+    </declare-styleable>
+
+    <declare-styleable name="AdyenCheckoutCustomTabs">
```

Comment:
Isn't this name supposed to be the same as a custom view? And since we don't have a view, is it necessary to declare a styleable?

Reply:
You're right I missed that, can you check the latest commit?